src/components/calculator/SavedDataSelector.tsx

```diff
@@ -42,20 +61,167 @@ const SavedDataLink: React.FunctionComponent<{
     </Button>
   )
 }
+
+// show [Build your own scenario] if no preset found
+const SavedScenarioContinue: React.FunctionComponent<{
+  variant: string
+  scenarioName: string
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+}> = (props): React.ReactElement => {
+  const { t } = useTranslation()
```

Comment:
move to global scope?

Reply:
./src/components/calculator/SavedDataSelector.tsx
  Line 23:15:  React Hook "useTranslation" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function  react-hooks/rules-of-hooks